frontend/test/__support__/e2e/helpers/e2e-misc-helpers.js

```diff
@@ -28,3 +28,14 @@ export function openNativeEditor(alias = "editor") {
     .as(alias)
     .should("be.visible");
 }
+
+/**
+ * Executes native query and waits for the results to load.
+ * Makes sure that the question is not "dirty" after the query successfully ran.
+ * @param {string} [xhrAlias ="dataset"]
+ */
+export function runNativeQuery(xhrAlias = "dataset") {
+  cy.get(".NativeQueryEditor .Icon-play").click();
+  cy.wait("@" + xhrAlias);
+  cy.icon("play").should("not.exist");
+}
```

Comment:
@nemanjaglumac I just copy-pasted this from [e2e-sql-filter-helpers](https://github.com/metabase/metabase/blob/6db73b0cfa743eefd2034ace9dd04d9b2c638d9f/frontend/test/metabase/scenarios/filters/helpers/e2e-sql-filter-helpers.js#L82) to `__support__/e2e/e2e-misc-helpers`, as it's very handy for the `native` test suite as well. Didn't like the idea to import helpers from another test group. Does it make sense to you? If so, I can follow up with a PR updating SQL filters tests to use this

Reply:
It's great that this happened.

There are going to be **many** helpers that are not tied to the specific folder only. I'm actually starting to think that ALL e2e helpers should live in the same place and be available to all tests. We should just import selectively.

Not sure how to approach the current situation you're in right now. Copy/pasting this helper into misc helpers doesn't seem like a good long term solution. We might end up duplicating a lot of the code and forgetting about it later.

In order to unblock you, feel free to proceed with this, and I'll deal with it later.
Thanks for the quick fix and great explanation of the problem.